<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>runTriangle</class>
 <widget class="QDialog" name="runTriangle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run TRIANGLE</string>
  </property>
  <widget class="Q3ButtonGroup" name="inputPolyFileGroup">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>283</y>
     <width>578</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Input a Poly File</string>
   </property>
   <widget class="QPushButton" name="polyFileBrowseButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>43</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="polyFileLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>44</y>
      <width>340</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PSLG File</string>
    </property>
   </widget>
  </widget>
  <widget class="Q3ButtonGroup" name="Options">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>578</width>
     <height>270</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QLineEdit" name="qOptionLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>122</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aOptionLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="otherOptionLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="otherOptionCheckBox">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>231</y>
      <width>130</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Other options</string>
    </property>
   </widget>
   <widget class="QLabel" name="aOptionLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>192</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Area (square meters)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="nOptionCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>26</y>
      <width>541</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>-n : Generate a list of triangle neighbours.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="VOptionsCheckBox">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>58</y>
      <width>541</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>-V : Verbose if checked. Quiet otherwise.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qOptionsCheckBox">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>92</y>
      <width>541</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>-q : Quality Mesh generation. A minimum angle may be specified.</string>
    </property>
   </widget>
   <widget class="QLabel" name="qOptionLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle (in Degree)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="aOptionsCheckBox">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>159</y>
      <width>541</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>-a : Apply a maximum triangle area constraint.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showOptionsButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>229</y>
      <width>20</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="Q3TextBrowser" name="MessageLog" native="true">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>438</y>
     <width>578</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>393</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Run!</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>393</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>393</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>QWidget</extends>
   <header>q3textbrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
